### DISCUSSION TITLE
Is it allowed to use svg or just html?

### DISCUSSION THREAD

**Discussion by coolgod** [cssbattle/forum#14] [Q&A] [answered]
Can I write svg tags?

**Comment by Esger** [comment, marked as answer]
Not anymore... just html now.

**Reply by gunnarbittersmann** [reply]
Actually, you can *write* SVG. That’s how I did the pens for [#57 Pillars](https://codepen.io/gunnarbittersmann/pen/rNWvOzb) and [#66 Batmicky](https://codepen.io/gunnarbittersmann/pen/xxRWxjO).
You can’t *submit* SVG.
(Well, as some of us know, you can. But the backend checks for SVG and returns a 0% match.)
